### PR TITLE
feat: consume new alby-js-sdk

### PR DESCRIPTION
**Pull request by pavanjoshi914**
### Describe the changes you have made in this PR

use types for account information response from alby-js-sdk 
new alby-js-sdk allows to use Error interface while catchinge error 
remove explicitly defined types
remove explict error types
add user agent

### Type of change

- `feat`: New feature (non-breaking change which adds functionality)

### Checklist

- [X] Self-review of changed code
- [X] Manual testing
- [X] Added automated tests where applicable
- [X] Update Docs & Guides
- For UI-related changes
- [X] Darkmode
- [X] Responsive layout

### CODE REVIEW

**Review by rolznz — APPROVED** [review]
Nice work @pavanjoshi914 !

File: src/extension/background-script/connectors/alby.ts [hunk 1]
```diff
@@ -224,6 +228,7 @@ export default class Alby implements Connector {
         client_id: clientId,
         client_secret: clientSecret,
         callback: redirectURL,
+        user_agent: "lightning-browser-extension",
```
Comment (rolznz) [inline src/extension/background-script/connectors/alby.ts:231]: @bumi could you please confirm this is the correct user agent?

Comment (bumi) [reply src/extension/background-script/connectors/alby.ts:231]: maybe we also add the version there. (which we could set in an environment variable / load from the package.json) 
https://github.com/getAlby/lightning-browser-extension/blob/7139fead6b9603ad9998cc4f28b5a15fd70ff401/webpack.config.js#L33-L35

Comment (pavanjoshi914) [reply src/extension/background-script/connectors/alby.ts]: resolved

**Pull request merged**